### PR TITLE
Add deprecated mark on Flake8 page

### PR DESCRIPTION
**Pull request by yykamei**
Python 2 will be retired soon.

https://www.python.org/psf/press-release/pr20191220/

### CODE REVIEW

File: docs/tools/python/flake8.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ hide_title: true
 | ----------------- | -------------------- | ----------------------- |
 | 3.7.9             | Python 2.7.16, 3.7.4 | http://flake8.pycqa.org |
```
Comment (ybiquitous) [inline docs/tools/python/flake8.md]: [IMO] This removal seems good:

```suggestion
| 3.7.9             | Python 3.7.4 | http://flake8.pycqa.org |
```

File: docs/tools/python/flake8.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ hide_title: true
...
 
+> **DEPRECATED**: Sider will drop the support of Python 2. So, the `version` option of `sider.yml` and the detection of Python version via `.python-version` features are going to be unavailable near the future.
```
Comment (ybiquitous) [inline docs/tools/python/flake8.md]: [suggestion] What about adding a link?

```suggestion
> **DEPRECATED**: Sider will drop the support of Python 2 (which will [be retired by April 2020](https://www.python.org/psf/press-release/pr20191220/)). So, the `version` option of `sider.yml` and the detection of Python version via `.python-version` features are going to be unavailable near the future.
```

**Comment by ybiquitous** [comment]
I've left some comments but not important! 😉

**Comment by yykamei** [comment]
Thank you :pray:

**Pull request merged**